bundle/manifests/nvidia-network-operator.clusterserviceversion.yaml

```diff
@@ -110,7 +110,7 @@ metadata:
   labels:
     operatorframework.io/arch.amd64: supported
     operatorframework.io/arch.arm64: supported
-  name: nvidia-network-operator.v24.10.0-rc.1
+  name: nvidia-network-operator.v24.10.0
```

Comment:
To update bundle, please run:
DEFAULT_CHANNEL=v24.10 CHANNELS=stable,v24.10 VERSION=24.10.0 TAG=nvcr.io/nvidia/cloud-native/network-operator@sha256:f258e8665d094c09159654759bc9ae41fbe0f7fffe95139ec02964759c3d8a87 make bundle